change config Path for more overview if you have a lot plugins installed

Joerg Bornkessel <hd_brummy@gentoo.org> 17 Feb 2007

diff -Naur vompserver-0.2.6.orig/bootpd.c vompserver-0.2.6/bootpd.c
--- vompserver-0.2.6.orig/bootpd.c	2007-02-17 18:25:39.300235973 +0100
+++ vompserver-0.2.6/bootpd.c	2007-02-17 18:26:03.418578373 +0100
@@ -99,7 +99,7 @@
 
   // Open a config file for the given MAC
 
-  const char* configDir = cPlugin::ConfigDirectory();
+  const char* configDir = cPlugin::ConfigDirectory( "vomp" );
 //  char* configDir = "/opt/dvb/vdr-config-1.3/plugins";
   if (!configDir)
   {
